package.json

```diff
@@ -42,8 +43,13 @@
     "@types/lodash": "^4.14.136",
     "@types/node": "^6.14.7",
     "@types/uuid": "^3.4.4",
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
+    "eslint": "6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "prettier": "^1.19.1",
```

Comment:
prettier is now on version `2`. Why are we in favor of using version `1`?

Reply:
Prettier 2 works only with >= 10